researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -654,6 +688,10 @@ private SchedulesAndTasksModel translateActivities(@NonNull ScheduledActivityLis
                 }
                 tsm.taskIsOptional = sa.getPersistent();
                 tsm.taskType = activity.getActivityType().toString();
+                if (sa.getFinishedOn() != null) {
+                    tsm.taskFinishedOn = sa.getFinishedOn().toDate();
+                }
+                tsm.taskGUID = sa.getGuid();
```

Comment:
Is it worth updating BridgeDataProviderTest.testLoadTasksAndSchedules() to check for these properties?

Reply:
It does seem like that method calls translateActivities so I'll add something in there to test this.

Reply:
Also added a test for this in testLoadTasksAndSchedules